std/net/curl.d

```diff
@@ -2328,11 +2338,21 @@ private bool decodeLineInto(Terminator, Char = char)(ref const(ubyte)[] basesrc,
   *     return len;
   * };
   * http.perform();
+  * ---
+  *
+  */
+
+/**
+  * HTTP Track progress
+  *
+  * Example:
```

Comment:
Probably remove the `Example:` lines as they don't look great in DDox.

Reply:
I removed all three, I'm not sure if that's what you meant.

Reply:
I think we should keep the first one.

Have a look at the generated docs in the DAutoTest CI status when it finishes, you can see the rendered old version and the new version with your changes there.